src/analysis/sampling/CHRR/chrrExpSampler.m

```diff
@@ -0,0 +1,125 @@
+function [samples,roundedPolytope] = chrrExpSampler(model,numSkip,numSamples,lambda,toRound,roundedPolytope,minFlux,maxFlux)
+% CHRRSAMPLER Generate uniform random flux samples with CHRR
+%   Coordinate Hit-and-Run with Rounding
+%
+% [samples,roundedPolytope,minFlux,maxFlux] = chrrSampler(model,numSkip,numSamples,toRound,roundedPolytope,minFlux,maxFlux);
+%
+%   chrrSampler will generate numSamples samples from model, taking
+%   numSkip steps of a random walk between each sample
+%
+%   Rounding the polytope is a potentially expensive step. If you generate multiple rounds
+%   of samples from a single model, you can save roundedPolytope from the first round and
+%   input it for subsequent rounds.
+%
+% INPUTS:
+% model ... COBRA model structure with fields:
+% .S ... The m x n stoichiometric matrix
+% .lb ... n x 1 lower bounds on fluxes
+% .ub ... n x 1 upper bounds on fluxes
+% .c ... n x 1 linear objective
+% numSkip ... Number of steps of coordinate hit-and-run between samples
+% numSamples ... Number of samples
+%
+% OPTIONAL INPUTS:
+% lambda ... the bias vector, i.e. generate samples from exp(-<lambda, x>) restricted to the feasible region.
+%            ***OPTIONAL ONLY IF model.c is nonzero, in which case we set
+%            lambda=model.c
+% toRound ... {0,(1)} Option to round the polytope before sampling.
+% roundedPolytope ... The rounded polytope from a previous round of
+%                     sampling the same model.
+% minFlux,maxFlux ... n x 1 flux minima and maxima from flux variability
+%                     analysis of the same model.
+%
+% OUTPUTS:
+% samples ... n x numSamples matrix of random flux samples
+% roundedPolytope ... The rounded polytope. Save for use in subsequent
+%                     rounds of sampling.
+%
+% November 2017, Ben Cousins and Hulda S. Haraldsdóttir

```

Comment:
please format the documentation according to the [documentation guide](https://github.com/opencobra/cobratoolbox/blob/master/docs/source/guides/documentationGuide.md)

Reply:
This should be fixed now.